moveit_ros/moveit_servo/src/servo_parameters.cpp

```diff
@@ -236,6 +422,19 @@ ServoParameters::SharedConstPtr ServoParameters::makeServoParameters(const rclcp
                         "greater than zero. Check yaml file.");
     return nullptr;
   }
+  return parameters;
+}
+
+ServoParameters::SharedConstPtr ServoParameters::makeServoParameters(const rclcpp::Node::SharedPtr& node,
+                                                                     const rclcpp::Logger& logger,
+                                                                     std::string ns, /* = "moveit_servo"*/
+                                                                     bool dynamic_parameters /* = true */)
+{
+  auto parameters = std::shared_ptr<ServoParameters>(new ServoParameters(logger, ns));
```

Comment:
Use make_shared here.  Reference: https://stackoverflow.com/questions/20895648/difference-in-make-shared-and-normal-shared-ptr-in-c